### PR TITLE
Created the scroll back to top button

### LINKED ISSUE #85: Create a scroll back to top button
I can create a scroll back to top button for the website, which can reach the home page of the website on click.
Can I make this contribution under Hacktoberfest'22 ?

### PR DESCRIPTION
**Pull request by zugzwang03**
close #85 

Screenshots:
scroll

### CODE REVIEW

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]
Hide the button for mobile view.
This doesn't look good at all 👇🏻👇🏻
Screenshot (675)

**Comment by zugzwang03** [comment]
Yeah fine, doing it soon

**Comment by SandeepKrSuman** [comment]
Also try playing around with the size. I guess a little smaller size would look good with the UI. Leaving this upto you!

**Comment by zugzwang03** [comment]
image
 how is this?

**Comment by SandeepKrSuman** [comment]
For larger screens 
```
width: 3.5rem
height: 3.5rem
```
is looking fine to me. What say?

File: src/components/MainSection/MainSec.css [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    position: fixed;
+    right: 4rem;
+    bottom: 4rem;
+    border-radius: 15px;
+    color: #ffffff;
+    background-color: #121916;
+    height: 5rem;
```
Comment (SandeepKrSuman) [inline src/components/MainSection/MainSec.css]:
```
height: 3.5rem;
width: 3.5rem;
```

**Comment by zugzwang03** [comment]
> For larger screens
> 
> ```
> width: 3.5rem
> height: 3.5rem
> ```
> 
> is looking fine to me. What say?

yeah done

File: src/components/MainSection/MainSec.css [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
+    background-color: #121916;
+    height: 3.5rem;
+    width: 3.5rem;
+    font-size: 1.3rem;
+    z-index: 999;
+}
+
+@media screen and (max-width:700px){
+    .top-btn{
+        opacity: 0;
```
Comment (SandeepKrSuman) [inline src/components/MainSection/MainSec.css:24]: making `opacity: 0` will only make it invisible but the element is present there everytime. 
Try display property instead.

```
display: none;
```

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]

**Comment by zugzwang03** [comment]
is it fine now?

**Review by SandeepKrSuman — APPROVED** [review]
LGTM 🚀

**Comment by SandeepKrSuman** [comment]
> is it fine now?

Looks good. Thanks!

**Pull request merged**